src/lib/protocols/proof/messages/messages.ts

```diff
@@ -0,0 +1,6 @@
+export enum MessageType {
+  PresentProof = ' "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/present-proof/1.0/propose-presentation"',
```

Comment:
```suggestion
  ProposePresentation = ' "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/present-proof/1.0/propose-presentation"',
```

Reply:
Done

Reply:
Doesn't seem resolved? By the way it has some weird double quotes and space in it
```suggestion
  ProposePresentation = 'did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/present-proof/1.0/propose-presentation',
```

Reply:
Resolved.